hellohilt/build.gradle

```diff
@@ -0,0 +1,51 @@
+apply plugin: "com.android.application"
+apply plugin: "kotlin-android"
+apply plugin: "kotlin-android-extensions"
```

Comment:
kotlin-android-extensions is deprecated now. Let's move to the parcelize plugin for all new things and we can remove it from the rest soon.

Reply:
Ah, yep. Did not see that.